### ISSUE TITLE
migrate to cljc

### ISSUE THREAD

**Issue by jiangts** [grammarly/perseverance#1]
it'd be great to have access to this in ClojureScript!

**Comment by alexander-yakushev** [comment]
Good idea! We haven't used it in our Clojurescript projects yet. Probably, it is time to start!
